ouroboros-network-framework/src/Ouroboros/Network/InboundGovernor/ControlChannel.hs

```diff
@@ -57,50 +56,43 @@ instance Show peerAddr
 
 -- | Server control channel.  It allows to pass 'STM' transactions which will
 -- resolve to 'NewConnection'.   Server's monitoring thread is the consumer
--- of this messages; there are two produceres: accept loop and connection
+-- of these messages; there are two producers: accept loop and connection
 -- handler for outbound connections.
 --
+-- GR-FIXME[D2]: Last sentence is DOU (see code-review-doc)
```

Comment:
@mtullsen where do you think it would be more natural to document who are the producers for this queue?

Reply:
I just realised that this is attached to the `ServerControlChannel` haddock, and it should be part of `ControlChannel` haddock.  Is that what you wanted to suggest?